docs/topics/braintree.rst

```diff
@@ -3,4 +3,23 @@
 Braintree
 #########
 
-More to come.
+Generate a token
+----------------
+
+Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate>`_:
+
+.. http:post:: /braintree/token/generate/
+
+    **Request**
```

Comment:
Yup can add. Reading the docs it felt like it was only used for the drop-in UI which we don't use.